anime_downloader/sites/helpers/request.py

```diff
@@ -48,7 +49,20 @@ def setup_func(url: str,
         referer : str
             a url sent as referer in request headers
         '''
-        sess = cf_session if cf else req_session
+        selescrape = 'dummy message'
+        if cf:
+            sess = cf_session
+        elif sel:
+            try:
+                from selenium import webdriver
+                from anime_downloader.sites.helpers import selescrape
+                sess = selescrape
+            except ImportError:
+                sess = cf_session
+                logger.warning("You can't use this provider because it requires selenium to work.\nIf you want to install it then run:  'pip install selenium' .")
```

Comment:
'You can't use' is false now. 'Provider may not work' is more correct

Reply:
ok